plute/text/utils/utterances.py

```diff
@@ -0,0 +1,89 @@
+"""
+This module provides functions that work on ASR alternatives described as:
+```
+[[{"transcript": "..."}]]
+```
+We call these alternatives_list.
+and
+```
+[{"transcript": "..."}]
+```
+as alternatives.
+"""
+import itertools
+from functools import reduce
+from operator import mul
+from typing import List, Optional
+from plute.types import Utterances, Transcript
```

Comment:
These two imports are a little inconsistent.

Let's have Utterance as a type and use List[Utterance] for rest.

Reply:
ok